cmd/xk6/main.go

```diff
@@ -160,6 +160,15 @@ func runBuild(ctx context.Context, args []string) error {
 		}
 	}
 
+	fmt.Println("***************************************************")
+	fmt.Println("* BUILD COMPLETE - FOLLOW THESE STEPS TO CONTINUE *")
+	fmt.Println("***************************************************")
+	fmt.Println("xk6 has now produced a new k6 binary with the following:")
+	for _, extension := range extensions {
+		fmt.Println(" -", extension.PackagePath)
```

Comment:
I wonder if this is necessary given that it was just provided as arguments. Especially as if `replace` and specific versions were used it might also be a bit confusing - not giving that information here as well.

Reply:
True. I was originally thinking to include version as well, but seemed like a potential rabbit-hole for little benefit. 😄